src/elements/content-preview/ContentPreview.js

```diff
@@ -1051,10 +1055,14 @@ class ContentPreview extends PureComponent<Props, State> {
      * @param {string} [version] - The version that is now previewed
      * @param {object} [additionalVersionInfo] - extra info about the version
      */
-    onVersionChange = (version?: BoxItemVersion, additionalVersionInfo?: Object): void => {
+    onVersionChange = (version?: BoxItemVersion, additionalVersionInfo?: AdditionalVersionInfo): void => {
         const { onVersionChange }: Props = this.props;
+        this.updateVersionToCurrent = getProp(additionalVersionInfo, 'updateVersionToCurrent');
```

Comment:
Is `getProp` needed here or can we just default the value in the function signature to an empty object?